core/src/main/java/com/oath/oak/OakMap.java

```diff
@@ -526,10 +522,10 @@ public OakRBuffer get(K key) {
             return m.internalOakMap.get(key);
         }
 
-        public void remove(K key) {
+        public boolean remove(K key) {
```

Comment:
Is it ZC API or regular ConcurrentNavigableMap interface?

Reply:
In the regular API, remove returns the previously associated value with the given key.
In the ZC API, I changed the signature to return ``boolean`` instead of plain ``void``